indico/modules/categories/blueprint.py

```diff
@@ -16,9 +16,47 @@
 
 from __future__ import unicode_literals
 
-from indico.modules.categories.controllers import RHCategoryStatistics
+from indico.modules.categories.controllers.display import RHDisplayCategory, RHCategoryStatistics
+from indico.modules.categories.controllers.management import RHCategorySettings
+from indico.web.flask.util import redirect_view, make_compat_redirect_func
 from indico.web.flask.wrappers import IndicoBlueprint
 
-_bp = IndicoBlueprint('categories', __name__, template_folder='templates', url_prefix='/category')
+from MaKaC.webinterface.rh import calendar, categoryDisplay
 
-_bp.add_url_rule('/<categId>/statistics', 'category-statistics', RHCategoryStatistics)
+_bp = IndicoBlueprint('categories', __name__, template_folder='templates', virtual_template_folder='categories',
+                      url_prefix='/category/<int:category_id>')
+
+_bp.add_url_rule('/manage/settings', 'manage', RHCategorySettings)
+#_bp.add_url_rule('/', 'display', RHDisplayCategory)
+_bp.add_url_rule('/', 'display', categoryDisplay.RHCategoryDisplay)
```

Comment:
is that actually used right now? it not i'd rather just leave the commented out line instead of adding this one - there's an entry in L41 from the legacy blueprint so it should work without this one.